app/src/main/java/io/github/droidkaigi/confsched/fragment/SessionDetailFragment.java

```diff
@@ -64,14 +65,35 @@ public void onCreate(@Nullable Bundle savedInstanceState) {
         AppUtil.setTaskDescription(activity, session.title, ContextCompat.getColor(activity, session.category.getVividColorResId()));
     }
 
+    // Receive all viewModel's properties changed.
+    private final Observable.OnPropertyChangedCallback onPropertyChanged = new Observable.OnPropertyChangedCallback() {
```

Comment:
I think you're not wrong.
But In ViewModel property's changes, using ``ObservableField`` and ``OnPropertyChanged``. 
It's common way in MVVM. 😄 

If it is possible, I want to write this:
```
activity.result.setBinding(viewModel.observableSession);
```

EventBus is useful, but take care.
Because it makes too much loose coupling.